drivers/storage/portworx/component/security.go

```diff
@@ -251,12 +251,12 @@ func (c *security) createPrivateKeysSecret(
 		},
 	}
 
-	err = k8sutil.CreateOrUpdateSecret(c.k8sClient, sharedSecret, nil)
+	err = k8sutil.CreateOrAppendToSecret(c.k8sClient, sharedSecret, nil)
```

Comment:
Not related to this change, but general question - When user is passing their own system and app secret as env variable, we will just end up creating a `px-system-secrets` secret and not using it, right?

Reply:
Answered offline - 
This is almost true. We always create and maintain `px-system-secrets`. We directly use the `px-system-secrets` for getting the apps secret for the operator to communicate w/ PX.